src/components/LabeledInput.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/require-default-props */
+import './LabeledInput.scss';
+
+type InputProps = {
+  name: string;
+  label:string;
+  value: string | number;
+  type?: string;
+  min?: number;
+  max?: number;
+  checked?: boolean;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void
+};
```

Comment:
```suggestion
type InputProps = {
  label:string;
} & React.HTMLProps<HTMLInputElement>;
```

we can just expand the input type element ([link](https://stackoverflow.com/questions/62180651/how-to-extend-jsx-types-for-typescript))